src/main/java/gregtech/integration/jei/recipe/RecipeMapCategory.java

```diff
@@ -106,15 +100,15 @@ public void setRecipe(IRecipeLayout recipeLayout, @Nonnull GTRecipeWrapper recip
                 if (handle.getItemHandler() == importItems) {
                     //this is input item stack slot widget, so add it to item group
                     itemStackGroup.init(handle.getSlotIndex(), true,
-                            new ItemStackTextRenderer(notConsumedItemInputs.get(handle.getSlotIndex())),
+                            new ItemStackTextRenderer(recipeWrapper.isNotConsumedInput(handle.getStack())),
```

Comment:
This is not working

Reply:
From just looking at the code (i have not tested this), I am assuming handle.getStack() is not returning the correct ItemStack - probably because the SlotWidget does not contain an item